ntc_templates/templates/index

```diff
@@ -755,6 +755,7 @@ huawei_vrp_display_http_server.textfsm, .*, huawei_vrp, di[[splay]] h[[ttp]] s[[
 huawei_vrp_display_mac-address.textfsm, .*, huawei_vrp, disp[[lay]] mac[[-address]]
 huawei_vrp_display_temperature.textfsm, .*, huawei_vrp, dis[[play]] tem[[perature]]
 huawei_vrp_display_nat_server.textfsm, .*, huawei_vrp, dis[[play]] na[[t]] ser[[ver]]
+huawei_vrp_display_ntp_status.textfsm, .*, huawei_vrp, dis[[play]] ntp status
```

Comment:
@brian-s-cao
Is it possible to run a shorter version of the command?

Look at the lines above this one for ideas and please verify at the CLI of a live device that a shorter command functions. _(If you have questions, please ask for clarification.)_